src/nodeHelper.ts

```diff
@@ -219,6 +220,16 @@ export class NodeHelper {
         return NodeHelper.getExtensionsProperty(op, NodeHelper.SPLIT_OPERATION_ORIGINAL_OPERATION, null);
     }
 
+    public static addSplitOperationSplittedOperations(op: Operation, splittedOp: Operation) {
```

Comment:
this name is confusing, will you be able to understand it 3 month later?

Reply:
Fixed. Thanks